components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/cmd/BuildCmd.java

```diff
@@ -59,18 +75,70 @@ public void execute() {
             return;
         }
 
+        projectName = GatewayCmdUtils.getProjectName(mainArgs);
+        projectName = projectName.replaceAll("[\\/\\\\]", "");
+        File projectLocation = new File(GatewayCmdUtils.getProjectDirectoryPath(projectName));
+
+        if (!projectLocation.exists()) {
+            throw new CLIRuntimeException("Project " + projectName + " does not exist.");
+        }
+
+        if(!isCompiled){
+            try{
```

Comment:
Add space before {. Fix spaces related errors in other places as well(ex: Line 86,115,126,135).

Reply:
code segment is no longer available.